lib/html-composer.js

```diff
@@ -104,9 +104,11 @@ function applyOptions(options, res) {
 
 /**
  * Render a template based on some generic data provided.
+ *
+ * @param {Object} res
  * @returns {Function}
  */
-function renderTemplate() {
+function renderTemplate(res) {
```

Comment:
kinda pendantic but I'd rather you pass in the `locals` property, since that's what you're really after.